sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 ## Release History
 
-### 2.11.0-beta.1 (Unreleased)
+### 2.11.0 (2024-04-26)
```

Comment:
we usually also mention the azure-cosmos version it upgrades to, but can do later

Reply:
```suggestion
### 2.11.0 (2024-04-26)
#### Other Changes
* Updated `azure-cosmos` to version `4.59.0`.
```